tests/test_iib_client.py

```diff
@@ -246,109 +302,67 @@ def test_iib_client_failure(fixture_build_details_json):
             )
 
 
-def test_iib_client_no_binary_image(fixture_build_details_json3):
```

Comment:
@lipoja this is not needed because the structure of attribute in models(add, rm, regenerate-bundle) is given. If the attribute model misses an attribute it fail.